lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -45,8 +45,8 @@ sub cleanup {
         record_info('Cleanup', "Executing $command cleanup");
         # 3 attempts for both terraform and ansible cleanup
         for (1 .. 3) {
-            my $cleanup_cmd_rc = qesap_execute(verbose => '--verbose', cmd => $command, cmd_options => '-d', timeout => 1200);
-            if ($cleanup_cmd_rc == 0) {
+            my @cleanup_cmd_rc = qesap_execute(verbose => '--verbose', cmd => $command, cmd_options => '-d', timeout => 1200);
```

Comment:
Same recommendation:
```
my ($cleanup_cmd_rc) = qesap_execute(verbose => '--verbose', cmd => $command, cmd_options => '-d', timeout => 1200);
```
Would have been better and prevented the change in L49, L58 & L59.